### PR TITLE
Changing routes to include api and updating edit form

### PR DESCRIPTION
**Pull request by aconsilvio**
Edit student form no longer updates page as you type and updates header and sidebar on submit

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -118,7 +118,7 @@ function isLoggedIn(req, res, next) {
   if (req.isAuthenticated())
     return next();
   // if they aren't redirect them to the home page
-  res.redirect('/login');
+  res.redirect('/api/login');
```
Comment (davidabrahams) [inline app.js]: We actually never end up using this middleware. We should keep it for when we actually start authenticating requests, just thought I'd mention it. For now, I'm going to change the redirect to just sending a `401` error.

File: public/javascripts/ng-apps.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ springInitiative.config(function($stateProvider, $urlRouterProvider, $locationPr
 
     /* Page views */
     .state('login', {
-      url: '/login',
+      url: '/api/login',
```
Comment (davidabrahams) [inline public/javascripts/ng-apps.js]: this should still probably just be /login

**Comment by davidabrahams** [comment]
Was this meant to be merged into master, @aconsilvio ?

**Comment by aconsilvio** [comment]
When I did it master was behind sass, so I thought it should merge to sass, but it doesn't matter at this point.

**Comment by davidabrahams** [comment]
Cool makes sense. Yeah once we're ready to merge we'll just merge it to master.

**Comment by davidabrahams** [comment]
I think this is ready to merge since Nora approved the other branch?

**Pull request merged**